locales/en/manage-kafka-permissions.json

```diff
@@ -50,6 +50,11 @@
     "resource_type":"Resource type",
     "resource_type_selected":"{{value}} Resource type"
   },
+  "resourcePrefix":{
+    "placeholder_text_starts_with":"Enter prefix",
+    "placeholder_text_is":"Enter name",
+    "prefix_aria_label":"Select resource prefix"
```

Comment:
A different aria-label should be used if the user is specifying a name. 

I'm pasting below a previous comment I provided that's related to this:
> Usually an aria-label should just repeat the visible text that displays adjacent to the input.
> ...
> In this case the aria-label I would recommend is something like this:
"{selected resource type} prefix" so that it would say "Topic prefix" or "Consumer group prefix", depending on the resource type they selected in the first column.

Is it possible to pass the selected resource type as part of the string here?
For example: 
* If user selects "Is" as the rule, then use "{resource type} name", e.g. "Topic name", "Consumer group name"
* If user selects "Starts with" as the rule, then use "{resource type} prefix", e.g. "Topic prefix", "Consumer group prefix"